pos_product_uom/models/product_template.py

```diff
@@ -0,0 +1,19 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    first_uom_category_id = fields.Many2one(comodel_name='uom.category', related='uom_id.category_id')
+    second_uom_id = fields.Many2one(comodel_name='uom.uom', string="Second Unit Of Mesure", domain="[('category_id', '=', first_uom_category_id), ('id', '!=', uom_id)]", help="List of secondary UOMs that can be used for this product")
+    is_use_second_uom = fields.Boolean(default=False)
```

Comment:
```suggestion
    is_use_second_uom = fields.Boolean()
```
Is there any need of explicitly setting its default value to False :thinking: ?

Reply:
No, it's not needed since Boolean fields default to False. 😅